cypress/e2e/01-organization_sign_up.cy.js

```diff
@@ -80,12 +80,8 @@ describe("Organization sign up", () => {
 
     cy.contains("Access data").click();
     cy.contains("Support admin");
-  });
-  it("navigates to the manage facilities page", () => {
```

Comment:
consolidating these it blocks into one flow per the best practice guidance from Cypress